pkgs/top-level/python-packages.nix

```diff
@@ -25835,11 +25835,11 @@ EOF
 
   pafy = buildPythonPackage rec {
```

Comment:
can you move this expression to `python-modules/pafy/default.nix` and use `fetchPypi`

Reply:
Done